notebooks/data-sources/gcsweb-ci/build-logs/build_log_EDA.ipynb

```diff
@@ -0,0 +1,1093 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">%matplotlib inline
</pre><p>Is no longer required to display plots with Jupyter</p><p> </p><p>Also add</p><p> </p><pre class="ql-syntax" spellcheck="false">import seaborn as sns
sns.set()
</pre><p> </p><p>We've found seaborn's default plots look a little nicer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Thanks. Updated.</p>